packages/message/src/index.vue

```diff
@@ -135,6 +124,8 @@ export default defineComponent({
     // Event handlers
     close() {
       this.closed = true
+      this.visible = false
+      this.onClose()
```

Comment:
Delete this line.

Reply:
'this.onClose()' can be deleted, but 'this.visible = false' cannot be deleted.

Reply:
What I said is `DELETE THIS LINE` which means the marked line 128. Thanks. I didn't use  plural here.